transport_nantes/transport_nantes/settings.py

```diff
@@ -73,6 +75,7 @@
     'asso_tn.middleware.default_context.DefaultContextMiddleware',
     'asso_tn.middleware.sessionCookie.SessionCookieMiddleWare',
     'utm.middleware.utm.UtmMiddleware',
+    'rollbar.contrib.django.middleware.RollbarNotifierMiddleware',
```

Comment:
You don't comment that this must be last.  You also add always even though we don't want to run rollbar in dev (uses credits).

Should this perhaps instead be
```
if ROLE in ("beta", "production"):
    # Rollbar wants to be last.  And we don't want it in dev.
    MIDDLEWARE.append('rollbar.contrib.django.middleware.RollbarNotifierMiddleware')
```

Reply:
Good catch